final portfolio available

rokiasissoko opened the pull request:
Take a look.

KevinWorkman committed:
Update portfolio-walkthrough.md

rokiasissoko committed:
Find my customized portfolio here.

rokiasissoko committed:
customized portfolio with images

rokiasissoko committed:
final portfolio available

tucosh commented on portfolio/src/main/webapp/contact.html:
```diff
@@ -0,0 +1,115 @@
+            <div class="row">
+                <div class="column">
+                    <a href="https://www.linkedin.com/in/rokiatousissoko/"> 
+                        <img src="/images/linked.png" style="width:75%"/>
```
Are these .png files included somewhere? I'm not seeing them ..

rokiasissoko replied:
Added the image files

tucosh reviewed (commented):
Where are the .png files located which you're referencing? Am I just overlooking them?

NAVG2000 committed:
Small typo in walkthrough 

Line was: "You'll also see examples that use more comlex server code next week!"

Typo fix change "comlex" to "complex"

nnooney commented:
We copied this code from the deployed site during a 1:1 to setup the GitHub commit flow. Rokia, can you also make a commit containing the images in your site on the branch RokiatouDevelopment and push the branch to GitHub? That will update this PR with the missing .png files and complete this Pull Request.

rokiasissoko committed:
Here'smy first ever servlet.

KevinWorkman committed:
Merge pull request #91 from NAVG2000/patch-1

Small typo in week-2-server/project-1-walkthrough.md

rokiasissoko committed:
Merge branch 'main' of https://github.com/google/software-product-sprint into main

rokiasissoko committed:
Servlet/Fetch/JSON completed for w2

rokiasissoko committed:
Merge branch 'main' into RokiatouDevelopment

rssissoko600 commented:
Added all the remaining image files and Week 2 work.

tucosh commented on portfolio/src/main/webapp/contact.html:
```diff
@@ -0,0 +1,111 @@
+    <div class="contactform">
+
+    <form id="contactform" method="post" action="#">
+      <div class="contanct-title">
```
I think this is a typo..

rokiasissoko replied:
Fixed this typo.

tucosh commented on portfolio/src/main/webapp/contact.html:
```diff
@@ -0,0 +1,111 @@
+        </br>
+        </br>
+
+        <button type="submit">Submit</button>
```
What's supposed to happen when I fill out the contact form and click Submit? (I tried, it basically redisplayed the page, but something weird happened with the stylesheet I guess, the graphic icons were distorted..)

rokiasissoko replied:
Working out implementing a confirmation page via Javascript, as of right now it should do nothing.

tucosh reviewed (approved):
It works! LGTM

nnooney commented on portfolio/src/main/java/com/google/sps/servlets/RandomServlet.java:
```diff
@@ -0,0 +1,43 @@
+   */
+  private String convertToJson(ArrayList<String> randoFact) {
+    String json = " ";
+    json+= "\"" + randoFact.get((int)Math.floor(Math.random() * randoFact.size())) + "\"";
```
nit: add a space between json and +=

rokiasissoko replied:
Added the space and deleted all commented out code. Thank you.

nnooney commented on portfolio/src/main/webapp/contact.html:
```diff
@@ -0,0 +1,111 @@
+
+    <nav>
+        <ul>
+            <!-- <li><a href="index.html">Home</a></li> -->
```
Generally it's good practice not to leave commented out code in a file. Can you remove this?

nnooney commented on portfolio/src/main/webapp/contact.html:
```diff
@@ -0,0 +1,111 @@
+                </div>
+            </div>
+          
+         <!-- <img src="/images/rokiaheadshot.jpeg" /> -->
```
same down here. Can you remove the commented code?

nnooney commented on portfolio/src/main/webapp/index.html:
```diff
@@ -7,8 +7,11 @@
     <script src="script.js"></script>
   </head>
   <body> 
+<<<<<<< HEAD
```
Whoops looks like there are some merge commits here. Can you resolve these?

nnooney reviewed (dismissed):
Looks good! I just left a few minor comments. Could you remove the commented out code? It's good to comment out sections when testing and debugging, but it's harder to read a project when there's a lot of commented out code, so it's good not to check in the commented code to git.
